doltpy_integrations/metaflow/examples/hospital_bounty/blog_post.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Introducing Dolt + Metaflow
+---
+
+## Background
+This post details how to use Metaflow with Dolt. [Metaflow](https://metaflow.org/) is a workflow manager that offers data scientists the ability to define small experiments and production jobs from a single API. [Dolt](https://docs.dolthub.com/) is a version controlled relational database. It provides a familiar SQL interface along with Git-like version control features for capturing state. Each commit corresponds to a complete state of the database at the time the commit was created. Both Dolt and Metaflow are open source.
```

Comment:
I think you need to mention in the opening that you want to use Dolt to get lineage, reproducibility, and back testing.